### DISCUSSION TITLE
How to dismiss Flyouts  on right mouse click

### DISCUSSION THREAD

**Discussion by Jhonfunk** [bdlukaa/fluent_ui#975] [Q&A] [answered]
### How to dismiss Flyouts  on right mouse click ?

**Comment by bdlukaa** [comment]
Any widget can be rendered inside a Flyout. You can use `GestureDetector` and listen to its `.onSecondaryTap`:

```dart
controller.showFlyout(
  ...,
  builder: (context) {
    return GestureDetector(
      onSecondaryTap: () {},
      child: ...,
    );
  },
);
```

**Reply by Jhonfunk** [reply]
@bdlukaa

Sorry, I didn't make myself clear. 

I meant that after the Flyouts pops up, How to dismiss Flyouts when right mouse button click on non-Flyouts area ?

**Reply by bdlukaa** [reply]
That is not currently possible, since it's not a common behavior. May I know the use case for this?

**Reply by Jhonfunk** [reply]
@bdlukaa 
Clicking the right mouse button to reposition the popup context menu is a habit of desktop users. I try to fit the user habits as much as possible.

**Reply by bdlukaa** [reply, marked as answer]
#983
